www/main.js

```diff
@@ -118,23 +127,25 @@ function readCookie(name) {
 function eraseCookie(name) {
     createCookie(name, "", -1);
 }
-function getRecentRoomNames() {
-  var roomArray = [readCookie("lastRoom"), readCookie("secondRoom"), readCookie("thirdRoom")];
-  for (var i = 0; i < roomArray.length; i++) {
-    if (roomArray[i] != 'undefined' && roomName !== null)
-        $('#lastRoom').append(" || <a href='/rooms/" + roomArray[i] + "'>#" + roomArray[i] + "</a>");
-    else
-        $('#lastRoom').append("");
-  }
+
+function getRecentRoomNames(currentRoomName) {
+    var $lastRooms = $('#lastRoom');
+    var roomArray = [readCookie("lastRoom"), readCookie("secondRoom"), readCookie("thirdRoom")];
+    for (var i = 0; i < roomArray.length; i++) {
+        if (roomArray[i] != 'undefined' && currentRoomName !== null)
```

Comment:
@Harxy I kept the reference to the current ```roomName``` when I updated this code but I'm not sure what this is for?